math/libtommath/Makefile

```diff
@@ -0,0 +1,112 @@
+############################################################################
+# apps/math/libtommath/Makefile
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+include $(APPDIR)/Make.defs
+
+CFLAGS += -Wno-format
+
+CSRCS = bn_cutoffs.c bn_deprecated.c bn_mp_2expt.c bn_mp_abs.c bn_mp_add.c bn_mp_add_d.c bn_mp_addmod.c \
```

Comment:
$(wildcard ...)?

Reply:
This is also a subset as far as I can tell.  Some files are conditioned.  I think here too it's better if we `wildcard` and then filter out those that we don't build or build conditionally.

Reply:
This is also a subset as far as I can tell.  Some files are conditioned.  I think here too it's better if we `wildcard` and then filter out those that we don't build or build conditionally.

Reply:
BTW, are the source extracted to this path directly?  No parent directory?

Reply:
There is a parent directory but we can just use the VPATH since it is flat.